core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -198,6 +198,12 @@
     z-index: 1;
   }
 
+  @media(max-width: 360px) {
```

Comment:
I wonder whether it's worth adding a brief comment to explain where the 360px comes from.